tests/components/axis/test_init.py

```diff
@@ -57,20 +58,15 @@
     device = await setup_axis_integration(hass)
     assert hass.data[axis.DOMAIN]
 
-    assert await axis.async_unload_entry(hass, device.config_entry)
+    assert await hass.config_entries.async_unload(device.config_entry.entry_id)
     assert not hass.data[axis.DOMAIN]
 
 
 async def test_populate_options(hass):
     """Test successful populate options."""
-    entry = MockConfigEntry(domain=axis.DOMAIN, data=ENTRY_CONFIG)
-    entry.add_to_hass(hass)
+    device = await setup_axis_integration(hass, options=None)
 
-    with patch.object(axis, "get_device", return_value=mock_coro(Mock())):
-
-        await axis.async_populate_options(hass, entry)
-
-    assert entry.options == {
+    assert device.config_entry.options == {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4925854)